fontir/src/ir.rs

```diff
@@ -4,6 +4,12 @@ use std::collections::BTreeMap;
 
 use serde::{Deserialize, Serialize};
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
+pub struct GlobalMetadata {}
```

Comment:
I wonder if we should call this global font-info stuff that cannot vary (e.g. upem, axes definitions) `StaticMetadata` to distinguish from other `VariableMetadata` properties that can vary across multiple masters like font-wide metrics ascender/descender and such for MVAR, which we haven't added yet as we're focussing on glyph-level variations for now, but eventually we will.

Reply:
+1, will rename

Reply:
...and rip off your comment as type documentation